main.py

```diff
@@ -359,54 +359,61 @@ def performEvaluation(issue_data) -> Result:
     specimin_path = get_specimin_env_var()
     specimin_command = build_specimin_command(repo_name, os.path.join(issue_folder_abs_dir, issue_id), issue_data[JsonKeys.ROOT_DIR.value], issue_data[JsonKeys.TARGETS.value], qual_jar_dir if os.path.exists(qual_jar_dir) else "")
     
-    if specimin_path is not None and os.path.exists(specimin_path):
-        result = run_specimin(issue_id ,specimin_command, specimin_path)
-    else:
-        result = run_specimin(issue_id ,specimin_command, os.path.join(issue_folder_abs_dir, specimin_project_name))
+    if specimin_path is not None and not os.path.exists(specimin_path):
+        print("Clone copy of Specimin is used")
+        specimin_path = os.path.join(issue_folder_abs_dir, specimin_project_name)
+    
+    result = run_specimin(issue_id ,specimin_command, specimin_path)   
     
     print(f"{result.name} - {result.status}")
 
-    return result  #TODO: will remove this line once preservation checking steps pass the CI pipeline from specimin
-    # build script is shipped with input program. It exists in the "specimin" directory of the input program's root directory.
-    # Coping the build script to the output directory of the minimized program.
-    build_script_path = os.path.join(issue_folder_abs_dir, issue_id, specimin_input, repo_name, specimin_project_name, "build.gradle")
 
-    if not os.path.exists(build_script_path): #TODO: when finish adding build script, raise exception to indicate missing build script
+    if not ("bug_type" in issue_data and issue_data["bug_type"]):
         return result
-    build_script_destination_path = os.path.join(issue_folder_abs_dir, issue_id, specimin_output, repo_name, "build.gradle")
 
-    copy_build_script = f"cp {build_script_path} {build_script_destination_path}"
-    subprocess.run(copy_build_script, shell=True)
+    # build.gradle and settings.gradle are shipped with input program. It exists in the "specimin" directory of the input program's root directory.
+    # Coping both to the output directory of the minimized program.
+    build_gradle_path = os.path.join(issue_folder_abs_dir, issue_id, specimin_input, repo_name, specimin_project_name, "build.gradle")
+    settings_gradle_path = os.path.join(issue_folder_abs_dir, issue_id, specimin_input, repo_name, specimin_project_name, "settings.gradle")
+
+    if not os.path.exists(build_gradle_path) or not os.path.exists(settings_gradle_path):
+        print(f"{issue_id} build.gradle or settings.gradle not found in the input program") 
```

Comment:
This should say "not found in the ./specimin/ directory of the input program" (or otherwise indicate exactly where it looks for the files).